Assets/Mona/TemplateScripts/PlayerProperties.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+namespace Mona
+{
+    [System.Serializable]
+    public class PlayerProperties
+    {   
+        [HideInInspector]
+        public int PropertiesVersion = 1;
+
+        // Fall speed of players in this volume
+        [Tooltip("Fall speed of players in this volume")]
+        public Vector3 Gravity = new Vector3(0, -15f, 0);
```

Comment:
Hey, did you test to see if the physics engine rotates the avatar with changes in gravity direction? If not, we might want to do without the `Vector3` - that was just a suggestion for that case where someone may want to create an [O'Neill Cylinder](https://astronomy.com/~/media/8F29E55B369C450D92B9FFF2A4D35B6D.jpg)-style effect or something like that.